doc/nrf/releases/release-notes-1.6.0.rst

```diff
@@ -0,0 +1,658 @@
+It includes the TF-M, MCUboot and the Zephyr RTOS open source projects, which are continuously integrated and re-distributed with the SDK.
+
+The |NCS| is where you begin building low-power wireless applications with Nordic Semiconductor nRF52, nRF53, and nRF91 Series devices.
+nRF53 Series devices are now supported for production.
+Wireless protocol stacks and libraries may indicate support for development or support for production for different series of devices based on verification and certification status.
+See the release notes and documentation for those protocols and libraries for further information.
+
+Highlights
+**********
+
+* Added production support for nRF5340 for Bluetooth LE in single protocol configuration and in multiprotocol operation with Thread for Bluetooth Peripheral role.
+* Added production support for multiple advertising sets to SoftDevice Controller.
+* Added features available in Trusted Firmware-M (TF-M), which is supported for development with nRF5340 and nRF9160.
+* Integrated Apple Find My, supported for production.
+* Added development support for the Pelion Device Management library for LTE-M/NB-IoT and Thread.
+* Added the :ref:`nrf_machine_learning_app` application for Thingy:52, which further integrates Edge Impulse Machine Learning with the |NCS|.
+* Added support for all Thread 1.2 mandatory features for routers and end devices supported for development for the nRF52 family.
+* Integrated more complete support for Matter (Project CHIP) which is supported for development for nRF52840 and nRF5340.
+* Integrated Apple HomeKit ADK v5.3 and added development support for nRF52832 and nRF52833 for Bluetooth LE HomeKit accessories.
+* The application layer part of ZBOSS Zigbee stack is available in source code now.
+* New sample :ref:`modem_shell_application` can be used to test various connectivity features.
+
+Release tag
+***********
+
+The release tag for the |NCS| manifest repository (|ncs_repo|) is **v1.6.0**.
+Check the :file:`west.yml` file for the corresponding tags in the project repositories.
+
+To use this release, check out the tag in the manifest repository and run ``west update``.
+See :ref:`cloning_the_repositories` for more information.
+
+Supported modem firmware
+************************
+
+See `Modem firmware compatibility matrix`_ for an overview of which modem firmware versions have been tested with this version of |NCS|.
+
+Use the latest version of the nRF Programmer app of `nRF Connect for Desktop`_ to update the modem firmware.
+See `Updating the nRF9160 DK cellular modem`_ for instructions.
+
+Known issues
+************
+
+See `known issues for nRF Connect SDK v1.6.0`_ for the list of issues valid for this release.
+
+Important known issues
+======================
+
+These known issues are potential regressions or high impact issues:
+
+* :ref:`SoftDevice Controller DRGN-15852 and DRGN-15852 <drgn_15852>`: Affects scanning on the nRF53 Series.
```

Comment:
It says *DRGN-15852 and DRGN-15852*, which is the same issue number (and root cause). Let's remove the duplicate:

```suggestion
* :ref:`SoftDevice Controller DRGN-15852 <drgn_15852>`: Affects scanning on the nRF53 Series.
```

Reply:
fixed